src/App.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import { Container } from '@material-ui/core'
 import { AuthProvider } from './components/auth/Auth'
-// import { Register } from './components'
-// import { Login } from './components'
+import PrivateRoute from './components/auth/PrivateRoute'
+import { Register } from './components'
```

Comment:
Since we're importing these into an index the register and login components can be imported on the same line

Reply:
Ah yes. I always forget that. I can make that change.

Reply:
Made this change.